includes/CreateWikiRegexConstraint.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Miraheze\CreateWiki;
+
+use MediaWiki\Logger\LoggerFactory;
+use StringUtils;
+
+class CreateWikiRegexConstraint {
+	/**
+	 * @param string $regex invalid regex to log for
+	 * @param string $name name of regex caller (config or message) to log for
+	 */
+	private static function logInvalidRegex( $regex, $name ) {
+		LoggerFactory::getInstance( 'CreateWiki' )->info(
+			'{name} contains invalid regex',
+			[
+				'name' => $name,
+				'regex' => $regex,
+			]
+		);
+	}
+
+	/**
+	 * @param array &$regexes
+	 * @param string $name name of regex caller (config or message) for logging
+	 * @param string $start
+	 * @param string $end
+	 */
```

Comment:
I think indicating this function has no return value is helpful to inform the difference to the usage of `array_filter()`.
```suggestion
	 * @return void
	 */
```

Reply:
Done. Thanks!